### PR TITLE
initial drafts for speakers

### PR DESCRIPTION
**Pull request by dcwalk**

### CODE REVIEW

File: sessions/submission-acceptance-form.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+
+# Recording
+
+- [ ] The presentation can be livestreamed
+- [ ] The presentation can be recorded and posted on the Our Networks website
+
+You can see an example of the previous recorded talks at: https://ournetworks.ca/recorded-talks/
+If you have any questions or concerns about recording and livestreaming contact us: orga@ournetworks.ca
+
```
Comment (benhylau) [inline sessions/submission-acceptance-form.md]: Add line about option to withdraw consent after talk is recorded and before publishing.

File: sessions/submission-acceptance.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Please let us know as soon as possible if you have any issues.]
+
+In order to confirm your participation, and even if you are no longer able to attend, please complete this form by Monday July 29, 2019: https://ournetworks.ca/session-confirmation
+
+The form asks you to confirm presenter details, as well as recording and honoraria preferences, You will be asked to confirm your session description in GitHub, please feel free to edit it as needed.
+
+We will launch registration along with the draft program by August 1, 2019. At that time we will have more details for presenters and special registration instructions at that time. 
+
+Many people indicated an interest in community billets, we are reaching out to people in Toronto and provide more details at registration launch. However, in the meantime we've prepared a travel page for those already thinking about accomodations:
```
Comment (garrying) [inline sessions/submission-acceptance.md]:
```suggestion
Many people indicated an interest in community billets, we are reaching out to people in Toronto and provide more details at registration launch. However, in the meantime we've prepared a travel page for those already thinking about accommodations: 
```

File: sessions/submission-acceptance.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+-
+- 
+
+Please let us know as soon as possible if you have any issues.]
+
+In order to confirm your participation, and even if you are no longer able to attend, please complete this form by Monday July 29, 2019: https://ournetworks.ca/session-confirmation
+
+The form asks you to confirm presenter details, as well as recording and honoraria preferences, You will be asked to confirm your session description in GitHub, please feel free to edit it as needed.
+
+We will launch registration along with the draft program by August 1, 2019. At that time we will have more details for presenters and special registration instructions at that time.
```
Comment (garrying) [inline sessions/submission-acceptance.md]:
```suggestion
We will launch registration along with the draft program by August 1, 2019. At that time we will have more details for presenters and special registration instructions. 
```

File: sessions/submission-acceptance-form.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+Please confirm attendance and pronunciation
+
+
+Pronunciation: 
+Preferred pronouns:
+
+
+# Recording
+
+- [ ] The presentation can be livestreamed
```
Comment (garrying) [inline sessions/submission-acceptance-form.md]:
```suggestion
- [ ] The presentation can be live streamed
```

File: sessions/submission-acceptance-form.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+- [ ] The presentation can be recorded and posted on the Our Networks website
+
+You can see an example of the previous recorded talks at: https://ournetworks.ca/recorded-talks/
+If you have any questions or concerns about recording and livestreaming contact us: orga@ournetworks.ca
```
Comment (garrying) [inline sessions/submission-acceptance-form.md]:
```suggestion
If you have any questions or concerns about recording and live streaming contact us: orga@ournetworks.ca
```

File: sessions/submission-acceptance-form.md (hunk 1 above)
Comment (dcwalk) [reply sessions/submission-acceptance-form.md]: added

**Comment by dcwalk** [comment]
Thanks for review! Added and PR in https://github.com/ournetworks/ournetworks.ca/pull/114

**Pull request merged**